packages/template-retail-react-app/app/components/product-view/index.jsx

```diff
@@ -220,7 +222,8 @@ const ProductView = forwardRef(
                     <Button
                         key="cart-button"
                         onClick={handleCartItem}
-                        disabled={showInventoryMessage}
+                        disabled={isBasketLoading || showInventoryMessage}
```

Comment:
@vcua-mobify from a UX standpoint, we can't have a `disabled` button state with no loading indicator or text that indicates that there is an in-progress action of some kind that the application is waiting for a response on

Reply:
A spinner will appear when the button is disabled. This is the same pre-existing mechanism we are currently using for the wishlist button as well

<img width="833" alt="Screenshot 2023-06-21 at 4 07 51 PM" src="https://github.com/SalesforceCommerceCloud/pwa-kit/assets/47404250/dd5a5b22-d033-4b70-a248-73e78a230bdd">

Reply:
Ok great. I'm good then.